s3sign/tests/test_views.py

```diff
@@ -62,17 +62,17 @@ def test_get_name_field(self):
         v = SignS3View()
         self.assertIsNotNone(v.get_name_field())
 
-    @override_settings(AWS_ACCESS_KEY='foo')
+    @override_settings(AWS_ACCESS_KEY_ID='foo')
     def test_get_aws_access_key(self):
         v = SignS3View()
         self.assertEqual(v.get_aws_access_key(), 'foo')
 
-    @override_settings(AWS_SECRET_KEY='foo')
+    @override_settings(AWS_SECRET_ACCESS_KEY='foo')
     def test_get_aws_secret_key(self):
         v = SignS3View()
         self.assertEqual(v.get_aws_secret_key(), 'foo')
 
-    @override_settings(AWS_UPLOAD_BUCKET='foo')
+    @override_settings(AWS_STORAGE_BUCKET_NAME='foo')
```

Comment:
In our setup, we tend to use a different S3 bucket for uploads, while all our assets are uploaded through django-storages to AWS_STORAGE_BUCKET_NAME. This is because the uploads bucket may be configured with different settings to allow for user upload, or possibly authentication on read.

The two other changes work for us, though.

Reply:
Hmmm, good point.